corehq/apps/export/static/export/js/download_export.ng.js

```diff
@@ -140,7 +140,20 @@
                 $('form[name="exportFiltersForm"]'));
             $scope.prepareExportError = null;
             $scope.preparingExport = true;
-            hqImport('analytix/js/kissmetrix').track.event("Clicked Prepare Export", {"Export type": $scope.exportList[0].export_type});
+            var userTypes = hqImport("hqwebapp/js/initial_page_data").get('user_types');
+            function getFilterName(exportType) {
+                return (exportType === "form" ? "emw" : "case_list_filter");
+            }
+            hqImport('analytix/js/kissmetrix').track.event("Clicked Prepare Export", {
+                "Export type": $scope.exportList[0].export_type,
+                "filters": _.map(
+                    $("#exportFiltersFormId input[name=" + getFilterName($scope.exportList[0].export_type) + "]")
```

Comment:
just a thought: based on jenny's post on the pr, this might be better as `$("#exportFiltersFormId").find("input[name=" + getFilterName($scope.exportList[0].export_type) + "]")`

Reply:
Doesn't find return the first matching value? I want to return a full array

Reply:
It should get all of them if I'm reading this right https://api.jquery.com/find/